exercises/largest-series-product/canonical-data.json

```
@@ -1,6 +1,6 @@
 {
   "exercise": "largest-series-product",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Would appreciate if another maintainer can chime in here and corroborate (or contradict) me here, but since some tracks actually use the error text here, I believe this should be a minor version bump, not a patch.
```suggestion
  "version": "1.3.0",
```

---

I re-read the [README](https://github.com/exercism/problem-specifications#test-data-versioning) and I agree, the version should be `"1.3.0"`.